### PR TITLE
To-Do-List-Peer-Review

### PR DESCRIPTION
**Pull request by lilyfrancis**
This is a To-Do-List project of Microverse.
In this project I added the following:

- Src folder in which includes HTML file, CSS file and JS file
- Dist folder in which include HTML, js, CSS files
- Webpack folder
- Linters config files

Below is the live link to the project
https://raw.githack.com/lilyfrancis/To-Do-List/add-and-remove/dist/index.html

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="write-task">
+      <input type="text" class="input-text" placeholder="Add your list ..." required>
+      <i class="far fa-plus-square"></i>
+    </div>
+    <div class="the-list"></div>
+      <button class="clean-completed">Clear All Completed</button>
+    </section>
+</body>
+
+</html>
```
Comment (tufoinnkuo10) [inline dist/index.html]: Great work I must say. Congratulations ㊗️ 😄 
never the less there is room for more improvement don't you think

Comment (tufoinnkuo10) [inline dist/index.html]: following the HTML good practice an extra free line is always required at the end of the code

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  color: rgb(70, 70, 70);
+}
+
+.fa-trash-alt {
+  position: absolute;
+  right: 0.5rem;
+  transition: width 2s;
+}
+
+.fa-trash-alt:hover {
```
Comment (tufoinnkuo10) [inline src/style.css:95]: The trash icon **(delete)** is hidden unless the mouse hovers on that task. i think that can be improved as well

**Review by tufoinnkuo10 — APPROVED** [review]
# STATUS: Changes Required  ♻️

### Hello @lilyfrancis   ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- You have one commit for everything. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏

almost

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/style.css (hunk 2 above)
Comment (lilyfrancis) [reply src/style.css:95]: Thanks for your review, I will make the requested changes

File: dist/index.html (hunk 1 above)
Comment (lilyfrancis) [reply dist/index.html]: Thanks for your review, I have done the requested changes, Please review

**Comment by tufoinnkuo10** [comment]
## Status: PR Approved 💯 🥇 💯 🥇 ✔
#### Hello @lilyfrancis, 

- [ ] The Pull Request has a proper title and description. 
- [ ] You have added a descriptive Readme file. Good Job.  
- [ ] There is no linter error is present. 
- [ ] Project has met all the technical aspects you needed for this project. Keep rocking!  

Your project is complete!  You have met all requirements for this project , your pull request message looks profesional . It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations once more! 🎉 👏👏🎉 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**